include/crm/crm.h

```diff
@@ -141,7 +141,7 @@ extern char *crm_system_name;
 #  define CRM_OP_LOCAL_SHUTDOWN 	"start_shutdown"
 #  define CRM_OP_SHUTDOWN_REQ	"req_shutdown"
 #  define CRM_OP_SHUTDOWN   PCMK_ACTION_DO_SHUTDOWN
-#  define CRM_OP_FENCE	 	"stonith"
+#  define CRM_OP_FENCE      PCMK_ACTION_STONITH
```

Comment:
Why not deprecate `CRM_OP_FENCE` in another commit?

Reply:
Huh, though it was used in other contexts. Added commit